### ISSUE TITLE
Error with some game not allow for decompile resource

### ISSUE THREAD

**Issue by hienngocloveyou** [RealAscarre/UE4-Mobile-Dumper#1]
Some game not allow for decompile resource file so can not compile manifest file.
Actually it allow we decompile resource file but when compile I got error like can not locate some resource file (ex: firebase resource)
So we can not continue do at step 8 "Copy the user permission from mod to game manifest file (Add read write to storage permission, some games don't use them but the dumper needs it)."

The solution is can you release a version without permission like LGL mod menu team ?
It will no need to insert permission in Manifest file, just do step 4 and we done.

Regards,

**Comment by AscarreX** [comment]
Yes actually I was not active for the past few years and I didn't get a look into the LGL Mod to bypass the System Alert Permission, I would update the project soon with all possible fixes and changes.

**Comment by AscarreX** [comment]
Okay so I looked again on this issue and found that editing the manifest is always possible and i have tested on multiple games and it worked every time for me. Don't decompile everything and use MT Manager or similar app to decompile only required resources like manifest and dex files.
